go/libkb/util.go

```diff
@@ -600,3 +600,13 @@ func MakeByte32(a []byte) [32]byte {
 	copy(b[:], a)
 	return b
 }
+
+func MakeByte32Soft(a []byte) ([32]byte, error) {
+	const n = 32
+	var b [n]byte
+	if len(a) != n {
+		return b, fmt.Errorf("MakeByte expected len %v but got %v slice", n, len(a))
+	}
+	copy(b[:], a)
+	return b, nil
+}
```

Comment:
Ah sorry, the function above used to be this, and I swapped it out because we didn't have any use cases yet. That said, in at least one of the cases where you're calling this, you are actually checking the length beforehand. Could drop that check maybe?